packages/scripts/index.js

```diff
@@ -44,6 +45,15 @@ const startHandler = async ({ file }) => {
     // Rollup will be emitting lifecycle events for the bundling process such as start, end and error. These events are
     // used to control a spinner in the terminal for each `input-output` bundle.
     switch (event.code) {
+      case 'END':
+        if (push) {
+          console.log();
+          console.log(chalk.gray.bold('yalc push'));
+          const yalcBin = path.resolve(__dirname, 'node_modules', '.bin', 'yalc');
```

Comment:
@christopherthielen, I've been thinking about this change. I'm wondering if there's a chance for `@spinnaker/scripts/node_modules/.bin/yalc` not to exist if `yalc` package is hoisted to the parent package's `node_modules`. I tried testing it locally and it didn't fail, but wondering when that hoisting might happen.

Reply:
It should still get a link in .bin even if it's hoisted, AFAIK